gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/aggregation/functions/AggregateTransactions.java

```diff
@@ -90,29 +86,29 @@ public GraphTransaction map(GraphTransaction graphTransaction) throws Exception
   /**
    * Applies the aggregate functions on the vertices of the given graph transaction.
    *
+   * @param aggregate map to hold the aggregate properties
    * @param graphTransaction graph transaction
    * @return final vertex aggregate value
    */
-  private Map<String, PropertyValue> aggregateVertices(GraphTransaction graphTransaction) {
-    Map<String, PropertyValue> aggregate = new HashMap<>();
-
+  private Map<String, PropertyValue> aggregateVertices(Map<String, PropertyValue> aggregate,
+                                                       GraphTransaction graphTransaction) {
     for (Vertex vertex : graphTransaction.getVertices()) {
-      aggregate = AggregateUtil.vertexIncrement(aggregate, vertex, vertexAggregateFunctions);
+      aggregate = AggregateUtil.increment(aggregate, vertex, vertexAggregateFunctions);
     }
     return aggregate;
   }
 
   /**
    * Applies the aggregate functions on the edges of the given graph transaction.
    *
+   * @param aggregate map to hold the aggregate properties
```

Comment:
This could be unclear too, "aggregate properties" sounds like some options for the aggregator to me, maybe put "aggregate values" instead